bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -366,9 +368,14 @@ Dimmer              iLR_dimmer                  "Dimmer [%.0f %%]"            (g
 
 Rollershutter       iLR_shutter                 "Shutter [%.0f %%]"           (gShutters, gLivingRoom) { channel="openwebnet:bus_automation:mybridge:LR_shutter:shutter", ga="Blinds", homekit = "Window" }
 
-Number:Power        iCENTRAL_Ta                 "Power [%.0f %unit%]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Ta:power" }
-Number:Power        iCENTRAL_Tb                 "Power [%.0f %unit%]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Tb:power" }
-
+Number:Power        iCENTRAL_Ta_pow              "Power [%.0f %unit%]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Ta:power" }
+Number:Power        iCENTRAL_Tb_pow              "Power [%.0f %unit%]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Tb:power" }
+Number:Energy       iCENTRAL_Ta_day              "Energy Day [%.1f]"                             { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Ta:totalizerDay" }
+Number:Energy       iCENTRAL_Tb_day              "Energy Day [%.1f]"                             { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Tb:totalizerDay" }
+Number:Energy       iCENTRAL_Ta_month            "Energy Month [%.1f]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Ta:totalizerMonth" }
+Number:Energy       iCENTRAL_Tb_month            "Energy Month [%.1f]"                           { channel="openwebnet:bus_energy_meter:mybridge:CENTRAL_Tb:totalizerMonth" }
```

Comment:
last parts of the channels are the old one

Reply:
ooh I am sorry! Fixed them